presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1079,6 +1080,11 @@ public SystemSessionProperties(
                         "Filter nulls from inner side of join",
                         featuresConfig.isOptimizeNullsInJoin(),
                         false),
+                booleanProperty(
+                        OPTIMIZE_NULLS_IN_JOINS_V2,
```

Comment:
Let's not proliferate the options. If there is an issue with the previous one let's fix it (or just remove it).

Reply:
I added a new option as a first-cut to make it easier for reviewers to pull this change and run comparisons themselves on old v. new while we build consensus on this approach. I will remove this flag now + delete the older implementation. Testing against older behavior can still be done by using a released Presto version